docs/user_guides/fs/storage_connector/creation/snowflake.md

```diff
@@ -16,18 +16,27 @@ When you're finished, you'll be able to query the database using Spark through H
 
 Before you begin this guide you'll need to retrieve the following information from your Snowflake account and database, the following options are **mandatory**:
 
-- **Snowflake Connection URL:** Consult the documentation of your target snowflake account to determine the correct connection URL. This is usually some form of your [Snowflake account identifier](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html). For example:
-
+- **Snowflake Connection URL:** Consult the documentation of your target snowflake account to determine the correct 
+  connection URL. There multiple formats supported by Snowflake. This is usually some form of your [Snowflake account 
+  identifier](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html) or an 
+[account URL](https://docs.snowflake.com/en/user-guide/organizations-connect.html#connecting-with-a-url), which can be retrieved from the Snowflake 
+  account home page or under the Accounts section under Admin settings. For example, using an account identifier format:
```

Comment:
I think I would link to this, you can either query it or in the new UI use Admin->Accounts to find the organization and account name:
https://docs.snowflake.com/en/user-guide/organizations-gs.html#label-viewing-organization-name

Reply:
you can also add this screenshot 
<img width="2004" alt="image" src="https://user-images.githubusercontent.com/8422705/192280813-d4ae66b5-8da5-43be-9d1f-062139b6cae6.png">

Reply:
done